.github/workflows/common.yaml

```diff
@@ -0,0 +1,27 @@
+name: Bigdata Commit Checks
+
+on: [pull_request]
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    # - name: Install devctl
```

Comment:
disabeling this for now until we get the Github_PAT token in this repo (not added by default to public ones).